ramlfications/types.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015 Ramlfications contributors
+
+from __future__ import absolute_import, division, print_function
+import attr
+import copy
+from six import MAXSIZE
+from six import iteritems
+from six import string_types
+
+from ramlfications.errors import UnknownDataTypeError
+from ramlfications.errors import DataTypeValidationError
+from ramlfications.parameters import Content
+from ramlfications.utils.common import OrderedDict
+from ramlfications.utils.parser import convert_caml_case
```

Comment:
yeah, we have a different policy with buildbot, I can use your policy if you whish

Doing one line per import allows to ease up a lot with the merge conflicts, when two people insert a new import from the same module.

We even have an automated tool for splitting the import in a precommit hook:
https://github.com/buildbot/buildbot/blob/master/common/fiximports.py